src/components/app/load-data.test.js

```diff
@@ -0,0 +1,74 @@
+import { getInitialState, loadData } from './load-data';
+import { saveState } from '../../utils';
+import formatData from '../../utils/format-data';
+import loremIpsum from '../../utils/data/lorem-ipsum.mock';
+import animals from '../../utils/data/animals.mock';
+import demo from '../../utils/data/demo.mock';
+
+describe('load-data', () => {
+  describe('getInitialState', () => {
+    const loremData = loadData('lorem');
+
+    it('returns an object', () => {
+      expect(getInitialState(loremData)).toEqual(expect.any(Object));
+    });
+
+    it('does not require the second argument', () => {
+      expect(getInitialState(loremData)).toEqual(
+        getInitialState(loremData, {})
+      );
```

Comment:
Not sure this is testing something of use i.e. you're just testing javascript initial parameter values works rather than some inner workings of the getInitialState method

Reply:
True, but I just wanted to add a test to guard against removing the default argument. Might be unnecessary 🤷‍♂